### ISSUE TITLE
Issues with >3 names ?

### ISSUE THREAD

**Issue by andyscott12** [mekwall/jquery-ui-tagging#3]
Hi There :)

Seems when you attempt to use more than 3 names - the plugin can't seem to handle it ? 

This is seen on this with - http://jsfiddle.net/mcWnL/150/

If you repeat the names >3 times - the plugin dies and crashes the browser ?

It is possible to update ? :)

**Comment by mekwall** [comment]
Hi! Yeah... I'm working on a new plugin that will extend jQuery UI Autocomplete instead. You can play with it here: http://jsfiddle.net/mekwall/sgxKJ/

**Comment by andyscott12** [comment]
@mekwall - hey thanks for the response :)

would be cool to have an option for a `<textarea>` and you can trigger a character anywhere in the `<textarea>` ? Also, testing in FF seems to yield a few bugs ?

**Comment by mekwall** [comment]
@andyscott12 Yeah I'm aware of the bugs and I'm going to work those out once I add it to github. Stay tuned!
